### PR TITLE
Remove alert on save, save template version in agreement and display it

### PR DESCRIPTION
**Pull request by aanil**
Also fix syntax bug when bulk update of bioinfo analysis fails.

### CODE REVIEW

File: run_dir/static/js/pricing_quote.js [hunk 1]
```diff
@@ -242,6 +250,9 @@ app.component('v-pricing-quote', {
             this.$root.fetchExchangeRates(sel_data['exchange_rate_issued_date'])
             this.loaded_version_desc = 'Version: '+ loaded_version + ' \n' +
                                     'Agreement_number: '+this.proj_id+'_'+timestamp_val
+            if('template_text' in sel_data){
```
Comment (sofiahag) [inline run_dir/static/js/pricing_quote.js:253]: Is there a project in dev that has a template text we can see?

File: run_dir/static/js/pricing_quote.js [hunk 2]
```diff
@@ -580,7 +592,7 @@ app.component('v-pricing-quote', {
               </div>
               <div class="card mt-5">
                 <div class="card-header">
-                  <h4>Preview</h4>
+                  <h4>Preview <small>Using template: {{this.template_text_data.doc_id}}-{{this.template_text_data.edition}}</small></h4>
```
Comment (sofiahag) [inline run_dir/static/js/pricing_quote.js:595]: Is this supposed to be centered? Maybe no data is being fetched locally, so it looks weird 🤷🏻‍♀️

File: status/bioinfo_analysis.py [hunk 3]
```diff
@@ -74,7 +74,7 @@ def post(self, project_id):
         neg_save_res = []
         for res in save_result:
             if not res[0]:
-                neg_save_res.append[res[1]]
+                neg_save_res.append(res[1])
```
Comment (sofiahag) [inline status/bioinfo_analysis.py:77]: Ah, but I get the `Not all changes saved in bulk update! Doc ids that failed to update are shown below` -error still 🤔 but maybe that's due to the connection error?

**Review by sofiahag — APPROVED** [review]

File: run_dir/static/js/pricing_quote.js (hunk 1 above)
Comment (aanil) [reply run_dir/static/js/pricing_quote.js:253]: P28959

File: run_dir/static/js/pricing_quote.js (hunk 2 above)
Comment (aanil) [reply run_dir/static/js/pricing_quote.js:595]: Its up on stage now.

File: status/bioinfo_analysis.py (hunk 3 above)
Comment (sofiahag) [reply status/bioinfo_analysis.py:77]: Hm it's just my user, maybe you need to be an admin?

Comment (aanil) [reply status/bioinfo_analysis.py:77]: You could use the genstat general user, it should have the required permissions.

**Pull request merged**